sdk/react/panels/audioView.js

```diff
@@ -155,22 +158,20 @@ class AudioView extends React.Component {
   };
 
   // MARK: - Volume
-
   getVolumeControlColor = () => {
-    if (!this.props.config.general.accentColor) {
-      if (!this.props.config.controlBar.volumeControl.color) {
+    if (this.props.config.general.accentColor) {
+      return this.props.config.general.accentColor;
+    } else {
+      if (this.props.config.controlBar.volumeControl.color) {
+        return this.props.config.controlBar.volumeControl.color;
+      } else {
```

Comment:
`else` is obsolete after `return`

Reply:
I know, it'll be fixed after fixing all warnings/errors from linter